tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -455,10 +458,11 @@ BenchmarkProblems:
         - LdsPadA: [-1]
         - LdsPadB: [-1]
         - 1LDSBuffer: [1]
+        - TransposeLDS: [0,1]
```

Comment:
TransposeLDS=0 for TN should be always rejected.
Would you like to add the test cases to make sure if it is rejected?

Reply:
Hi Koji, I've tested before without the parts at line 1921 and it didn't reject. So that's why I added the part at line 1921.

Reply:
Hi Koji. I saw your another comment to remove the **tc == 'B'** at line 1927. Yes, that would work. So I guess I can just remove that comment about "**(B only)**", which means both dtvA, B reject those cases.

Reply:
Done with removing 'tc == B'